.github/workflows/brakeman.yml

```diff
@@ -0,0 +1,13 @@
+name: Brakeman
```

Comment:
is this just the default config from running `brakeman -C` ?

Reply:
I think this is just the name of the GH workflow